app/ui/browser/views/browser.jsx

```diff
@@ -11,6 +11,7 @@ specific language governing permissions and limitations under the License.
 */
 
 import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
```

Comment:
There's no motiviation for this change in the commit comment.  Can you explain why this is better?